### ISSUE TITLE
Add supersingular primes 

### ISSUE THREAD

**Issue by jvoight** [LMFDB/lmfdb#2323]
Seems like this would be easy to add in genus 1 and useful to see, up to a reasonable bound.  The bound may be just the number of a_pp's that are stored in the L-function?

In genus 2, maybe we just list the superspecial (or supersingular) primes; I suppose the case p-rank 1 is also interesting?

**Comment by JohnCremona** [comment]
Right now the database only contains ap for the first 25 primes (p<100) for elliptic curves over Q, which will not include many supsersingular primes.  Of course it would be easy to increase this number.  At the moment they are not used for much: only for when we have p-adic regulator data (quite small conductor and positive rank), in order to dcide which primes to display on a page such as http://beta.lmfdb.org/EllipticCurve/Q/37/a/1 (in the drop-down "Choose a prime" list).

**Comment by jvoight** [comment]
Oh, well, a one-off computation to give supersingular primes < 1000 (or another bound) seems easy to do.  Should we store the a_p's the come from this, or not?

**Comment by JohnCremona** [comment]
Probably sufficient just to store the primes, since the ap can easily be recomputed, and will anyway be 0 for p>3 (genus 1!).  I can imagine people wanting to search on which primes are supersingular.

What about CM curves?

I also saw that for elliptic curves over number fields, currently we store no ap at all.  Another job for someone.

**Comment by AndrewVSutherland** [comment]
I would suggest just storing and displaying the first n supersingular/superspecial primes up to some fixed bound N, say n=5 or 10 and N=10^9.  Then we can treat all curves the same way.  I agree that it is not necessary to store the ap's (or more generally the L-polynomials).

**Comment by edgarcosta** [comment]
I agree with Drew's idea. For CM, we should also add the note to say that
the set of supersingular primes is the set of ramified or inert primes in
Q(\sqrt{-d}).

**Comment by JohnCremona** [comment]
Would it not be better to sore the supersingular primes up to a fixed bound rather than limit the number (as well or instead)?  Say we store the first 5, someone searches for curves with 83 supersingular but with 5 smaller ss primes, then the curve would not be found.
With N=10^3 there are ~5000 isogeny classes (of elliptic curves over Q in the database) with no ss primes less than N, so that is too small.  Perhaps N=10^4 or 10^5 is sufficient.

**Comment by edgarcosta** [comment]
Looking back at this. I would be more flexible and would store at least n supersingular primes, and all the supersingular primes up to somebound (if the curve is not CM).

**Comment by kedlaya** [comment]
This came up again in the context of hypergeometric motives. I would expect to see all primes up to some bound, for the reason that John gave. For N = 10^5, what is the maximal number of ss primes less than N for a single non-CM curve in the database?

**Comment by edgarcosta** [comment]
Asymptotically, Lang--Trotter predicts
```
 #{ p <= N : p is supersingular } ~ C_E Sqrt(N)/log(N)
```
There is also an expression for `C_E`, but I am not sure how helpful would it be in this context, see Katz - Lang-Trotter revisited.
